File: integrations/shipstation/lib/phoenix/shipping_method.go
package phoenix

type ShippingMethod struct {
	ID                    int    `json:"id"`
	ShippingMethodID      uint   `json:"shipping_method_id"`
	AdminDisplayName      string `json:"admin_display_name"`
	StorefrontDisplayName string `json:"storefront_display_name"`
	Price                 int    `json:"price"`
}

File: ashes/src/paragons/shipping-method.js
/* @flow */

export type ShippingMethod = {
  id: number,
  adminDisplayName: string,
  storefrontDisplayName: string,
  code: string,
  price: number,
  isActive: bool,
};

export type CreatePayload = {
  adminDisplayName: string,
  storefrontDisplayName: string,
  code: string,
  price: number,
};

export type UpdatePayload = {
  adminDisplayName?: string,
  storefrontDisplayName?: string,
  price?: number,
};
